src/modules/Map/InteractiveMap/components/Legend/Legend.js

```diff
@@ -51,7 +51,7 @@ export const Legend = ({
           color,
           items: subItems,
           shape = 'square',
-          radius = DEFAULT_RADIUS,
+          diameter = DEFAULT_DIAMETER,
```

Comment:
I think you should keep backward compatibility by still allowing to have a *radius* parameter. This way, your PR won't be a breaking change anymore.

So you keep your changes about renaming and using *diameter*, but you can adjust so the app won't crash if provided parameter is still *radius*.

```suggestion
          radius,
          diameter = radius || DEFAULT_DIAMETER,
```

Also, keep unit tests for both diameter AND radius params.